SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1220,6 +1216,16 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 			   *Actor->GetName(), Op.request_id, Op.entity_id, UTF8_TO_TCHAR(Op.message));
 		break;
 	}
+
+	if (static_cast<Worker_StatusCode>(Op.status_code) == WORKER_STATUS_CODE_SUCCESS && Actor->IsA<APlayerController>())
```

Comment:
Any reason this is not folded into the switch case for `WORKER_STATUS_CODE_SUCCESS` above?

Reply:
Hazarding a guess of how @alastairdglennie's mind works, but probably because the switch statement is just responsible for logging generic state updates, whereas this is a special case of only certain actors types, and executions functional logic.

Reply:
Yeah that is fair, on second thought I would suggest maybe generalising / breaking out the logging code a bit, but that seems out of scope.